internal/clients/nightfall/nightfall.go

```diff
@@ -278,3 +284,34 @@ func (n *Client) ReviewDiff(
 
 	return comments, nil
 }
+
+func filterFileDiffs(fileDiffs []*diffreviewer.FileDiff, fileIncludeList, fileExcludeList []string) []*diffreviewer.FileDiff {
+	filteredFileDiffs := fileDiffs
+	if fileIncludeList != nil && len(fileIncludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileIncludeList, true)
+	}
+	if fileExcludeList != nil && len(fileExcludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileExcludeList, false)
+	}
+	return filteredFileDiffs
+}
+
+func filterByFilePath(fileDiffs []*diffreviewer.FileDiff, regexPatterns []string, include bool) []*diffreviewer.FileDiff {
+	filteredFileDiffs := []*diffreviewer.FileDiff{}
+	for _, fd := range fileDiffs {
```

Comment:
if this is for file inclusion, append to `filteredFileDiffs` if the filename matches a regex pattern `or` if this is for file exclusion, append to `filteredFileDiffs` if the filename doesn't match any of the regex patterns